docs/source/markdown/podman-stats.1.md

```diff
@@ -30,17 +30,33 @@ Pretty-print container statistics to JSON or using a Go template
 
 Valid placeholders for the Go template are listed below:
 
-| **Placeholder** | **Description**    |
-| --------------- | ------------------ |
-| .ID             | Container ID       |
-| .Name           | Container Name     |
-| .CPUPerc        | CPU percentage     |
-| .MemUsage       | Memory usage       |
-| .MemUsageBytes  | Memory usage (IEC) |
-| .MemPerc        | Memory percentage  |
-| .NetIO          | Network IO         |
-| .BlockIO        | Block IO           |
-| .PIDS           | Number of PIDs     |
+| **Placeholder**     | **Description**                                          |
+| ---------------     | ------------------                                       |
+| .ID                 | Container ID (truncated)                                 |
+| .ContainerID        | Container ID (full hash)                                 |
+| .Name               | Container Name                                           |
+| .CPUPerc            | CPU percentage                                           |
+| .MemUsage           | Memory usage                                             |
+| .MemUsageBytes      | Memory usage (IEC)                                       |
+| .AvgCPU             | Average CPU (full precision float)                       |
+| .AVGCPU             | Average CPU (formatted)                                  |
+| .CPU                | Percent CPU (full precision float)                       |
+| .CPUNano            | CPU Usage, total, in nanoseconds                         |
+| .CPUSystemNano      | CPU Usage, kernel, in nanoseconds                        |
+| .Duration           | Same as CPUNano (FIXME: why?)                            |
+| .MemLimit           | Memory limit, in bytes                                   |
+| .MemPerc            | Memory percentage                                        |
+| .NetIO              | Network IO                                               |
+| .BlockIO            | Block IO                                                 |
+| .PIDS               | Number of PIDs                                           |
+| .NetInput           | Network Input                                            |
+| .NetOutput          | Network Output                                           |
+| .BlockInput         | Block Input                                              |
+| .BlockOutput        | Block Output                                             |
+| .PIDs               | Number of PIDs                                           |
+| .SystemNano         | Current system datetime, nanoseconds since epoch         |
+| .UpTime             | WTF? This is Duration (CPUNano) in human-readable form?? |
+| .Up                 | Same as UpTime (FIXME: why?)                             |
```

Comment:
Two FIXMES and one WTH where I'd appreciate feedback

Reply:
UpTime looks like it is the value in /proc/uptime. Which RHEL describes as
`total number of seconds the system has been up`.
In Containers (Really PID Namespaces) this would be `the total number of seconds the container has been up`

I don't see '{{.Up so I don't think we should document it.

Reply:
>I don't see '{{.Up so I don't think we should document it.

Well, that's just the problem in a nutshell. Where are you looking, that you don't see `.Up`? Because I guarantee that it exists, and this is exactly the problem I'm trying to address.

First: it exists:
```console
$ bin/podman stats --no-reset --no-stream --format '{{.Up}}'
57.940104s
```
Second, and much more importantly, command-completion offers it as a suggestion:
```console
$ bin/podman stats --format '{{.'
{{.AVGCPU}}         {{.CPUSystemNano}}  {{.MemPerc}}        {{.PIDS}}         
{{.AvgCPU}}         {{.CPU}}            {{.MemUsageBytes}}  {{.PIDs}}         
{{.BlockIO}}        {{.ContainerID}}    {{.MemUsage}}       {{.PerCPU}}       
{{.BlockInput}}     {{.ContainerStats.  {{.Name}}           {{.SystemNano}}   
{{.BlockOutput}}    {{.Duration}}       {{.NetIO}}          {{.UpTime}}       
{{.CPUNano}}        {{.ID}}             {{.NetInput}}       {{.Up}}           <<<<<<<<<<<<<<
{{.CPUPerc}}        {{.MemLimit}}       {{.NetOutput}}
```
Therefore, IMO, it should be documented. This is what I'm trying to do in #14046. Nobody has stepped in to document the FIXMEs in that PR, so I'm taking baby steps myself. There are some fields, like this one, that are way above my ability to explain.

Reply:
I got that from reading the code, but perhaps there is other mechanisms for setting it?

```
In Main I now see:
$ ./bin/podman stats --format {{.
{{.AvgCPU}}         {{.BlockIO}}        {{.ContainerStats.  {{.CPUPerc}}        {{.ID}}             {{.MemUsage}}       {{.NetInput}}       {{.PerCPU}}         {{.SystemNano}}     
{{.AVGCPU}}         {{.BlockOutput}}    {{.CPU}}            {{.CPUSystemNano}}  {{.MemLimit}}       {{.MemUsageBytes}}  {{.NetIO}}          {{.PIDs}}           {{.Up}}             
{{.BlockInput}}     {{.ContainerID}}    {{.CPUNano}}        {{.Duration}}       {{.MemPerc}}        {{.Name}}           {{.NetOutput}}      {{.PIDS}}           {{.UpTime}}         
IN Release I see
$ /bin/podman stats --format {{.
{{.AvgCPU}}         {{.BlockOutput}}    {{.CPU}}            {{.CPUSystemNano}}  {{.MemLimit}}       {{.MemUsage}}       {{.NetInput}}       {{.PerCPU}}         {{.SystemNano}}     
{{.BlockInput}}     {{.ContainerID}}    {{.CPUNano}}        {{.Duration}}       {{.MemPerc}}        {{.Name}}           {{.NetOutput}}      {{.PIDs}}           {{.UpTime}}     
```

Reply:
Yes, @Luap99 has been working diligently to expand the fields shown by completion. See #14330 and #14263 and #14216

Reply:
Sure. and I like the idea.  The problem is we have leaked into this area fields we want users to use, along with fields we don't want them to use, and up to now, it was very unlikely they would discover the ones that leaked out.  Which is why I don't want to document the mistake ones.